dataland-frontend/tests/component/components/resources/frameworkDataSearch/sme/SmePanel.cy.ts

```diff
@@ -1,164 +0,0 @@
-import { type FixtureData, getPreparedFixture } from "@sharedUtils/Fixtures";
-import {
-  type DataMetaInformation,
-  type SmeData,
-  type CompanyAssociatedDataSmeData,
-  type DataAndMetaInformationSmeData,
-  DataTypeEnum,
-  PercentRangeForInvestmentsInEnergyEfficiency,
-  PercentRangeForEnergyConsumptionCoveredByOwnRenewablePower,
-} from "@clients/backend";
-import { minimalKeycloakMock } from "@ct/testUtils/Keycloak";
-import SmePanel from "@/components/resources/frameworkDataSearch/sme/SmePanel.vue";
-import { threeLayerTable } from "@sharedUtils/components/ThreeLayerTable";
-import { QaStatus } from "@clients/qaservice";
-import { assertDefined } from "@/utils/TypeScriptUtils";
-import { smeDataModel } from "@/components/resources/frameworkDataSearch/sme/SmeDataModel";
-
-describe("Component tests for SmePanel", () => {
-  let preparedFixtures: Array<FixtureData<SmeData>>;
-  const companyId = "mock-company-id";
-
-  before(function () {
-    cy.fixture("CompanyInformationWithSmePreparedFixtures").then(function (jsonContent) {
-      preparedFixtures = jsonContent as Array<FixtureData<SmeData>>;
-    });
-  });
-
-  it("Check Sme view page for company with one Sme data set", () => {
```

Comment:
There are still sme enums in model/enums/sme in the backend.

Reply:
Eieiei.      Sorry und gut gesehen.

Habs entfernt.